README.md

```diff
@@ -1,91 +1,84 @@
-
-**jDataView** provided a polyfill for the **DataView API** with own convenient extensions using the best available option between Arrays, TypedArrays, NodeJS Buffers and DataViews.
-
-Now that `DataView` is natively available in all engines, **jDataView 3** acts as a layer on top of it with powerful methods for dealing with non-standard binary data types, such as strings and arbitrary-sized integers.
+## [Documentation](https://github.com/jDataView/jDataView/wiki)
+jDataView extends the built-in JavaScript `DataView` class, so all of [it's documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DataView) is applicable here. 
 
-Also check out ([jBinary](https://github.com/jDataView/jBinary))
-========================
+### API Reference
 
-For complicated binary structures, it may be hard enough to use only low-level get/set operations for parsing,
-processing and writing data.
+  * [jDataView constructor](https://github.com/jDataView/jDataView/wiki/jDataView-constructor)
+  * jDataView Extensions
+    * [Operation control](https://github.com/jDataView/jDataView/wiki/Operation-control)
+    * [writeXXX methods](https://github.com/jDataView/jDataView/wiki/writeXXX-methods)
+    * [Strings and Blobs](https://github.com/jDataView/jDataView/wiki/Strings-and-Blobs)
+    * [Bitfields](https://github.com/jDataView/jDataView/wiki/Bitfields)
+    * [Internal utilities](https://github.com/jDataView/jDataView/wiki/Internal-utilities)
 
-In addition, most likely you might need convenient I/O methods for retrieving data from external sources such like
-local files (using File API or from Node.js), remote files (via HTTP(S)), data-URIs, Node.js streams etc. as well
-as for displaying generated content to user on webpage in image/video/audio/... containers
-or even as simple download link.
 
-If you faced any of these problems, you might want to check out new [jBinary](https://github.com/jDataView/jBinary)
-library that works on top of **jDataView** and allows to operate with binary data in structured and convenient way.
+## History
 
-Demos
-=====
+Check out the original [jDataView article](https://blog.vjeux.com/2011/javascript/jdataview-read-binary-file.html) by [Vjeux](https://github.com/vjeux), jDataView's original author.
 
-[HTTP Live Streaming realtime converter and player demo](http://rreverser.github.io/mpegts/) implemented using [jBinary](https://github.com/jDataView/jBinary) data structures.
-[![Screenshot](http://rreverser.github.io/mpegts/screenshot.png?)](http://rreverser.github.io/mpegts/)
+There are three ways to read a binary file from the browser.
 
----
+* The first one is to download the file through XHR with [charset=x-user-defined](https://developer.mozilla.org/en/using_xmlhttprequest#Receiving_binary_data). You get the file as a **String**, convert it to byte **Array** and you have to rewrite all the decoding and encoding functions (getUint16, getFloat32, ...). All the browsers support this.
```

Comment:
TBH I think we can remove the historical section from the README. Maybe move it to a wiki page, if it's for a historical preservation interest, but at this point it would be only distracting in main docs for new users.